e2e/wdio/core/fixtures/appData/date-picker-contents.ts

```diff
@@ -0,0 +1,32 @@
+export const currentMonth = new Date().getMonth() + 1;
+export const currentYear = new Date().getFullYear().toString();
+export const currentMonthWithZero = ('0' + `${currentMonth}`).slice(-2);
+export const firstSimpleDatePickerIndex = 0;
+export const secondSimpleDatePickerIndex = 1;
+export const rangeDatePickerIndex = 2;
+export const internationalizationIndex = 3;
+export const firstFormatterIndex =  4;
+export const secondFormatterIndex = 5;
+export const nullValidityIndex = 6;
+export const reactiveFormIndex = 7;
+export const reactiveFormRoDIndex = 8;
+export const reactiveFormDisableIndex = 9;
+export const reactiveFormDisableRoDIndex = 10;
+export const positionIndex = 11;
+export const secondSimpleDatePickerDate = `${currentMonth}/1/2021`;
+export const rangeDatePicker = `${currentMonth}/1/2021 - ${currentMonth}/15/2021`;
+export const internationalizationDate = `01/${currentMonthWithZero}/2021`;
+export const internationalizationGerman = `1.${currentMonth}.2021`;
+export const internationalizationBulgarian = `1.${currentMonthWithZero}.2021 г.`;
+export const firstFormatterDate = `${currentMonthWithZero}/01/21`;
+export const secondFormatterDate = `${currentMonthWithZero}/01/21 to ${currentMonthWithZero}/15/21`;
+export const reactiveFormRoDDate = '10/1/2020 - 10/15/2020';
+export const reactiveFormDisableDate = `${currentMonth}/${new Date().getDate().toString()}/2021`;
+export const reactiveFormDisableRoDDate = `${currentMonth}/${new Date().getDate().toString()}/2021 - ${currentMonth}/${new Date().getDate().toString()}/2021`;
+export const date = `${currentMonth}/1/2021`;
+export const text = 'test';
+export const highlightedColor = ['rgb(8,84,160)', 'rgba(8,84,160,1)'];
```

Comment:
hardcoded color values

Reply:
https://github.com/SAP/fundamental-ngx/pull/4935#discussion_r607775724

Reply:
Can you delete this?